api/doc/2017/dev-cycle.md

```diff
@@ -1,43 +0,0 @@
-# Failed Attempt to Improve Development Cycle Time
-
```

Comment:
Just wanted to confirm if you deliberately deleted  this file rather than moving to docs folder